docs/requirements.txt

```diff
@@ -1,3 +1,5 @@
--r https://github.com/ansible/ansible/raw/devel/docs/docsite/requirements.txt
+# we need 0.17.0 at least, and that's not released yet
+git+https://github.com/ansible-community/antsibull
```

Comment:
Maybe they find a better name until then...
Isn't there an `ansibullbot` around somewhere?

Reply:
Yes, I don't like the name too much either, but it's not for us to decide and I'm really happy they split out the tooling in the first place.

Reply:
We had long discussions about that name, and in the end we decided on this one since a) the name should not be `ansible-*` (that's reserved to ansible core projects), and b) it should not make tab completion harder (originally we had ansibulled, and that made tab completing both ansible and ansibulled harder). antsibull satisfies both, and is also similar enough to ansible (depending on your English pronounciation, it's very similar ;) ).

Out of curiosity: why do you dislike the name antsibull?

Reply:
I type ansibull all the time

Reply:
That sometimes happens to me too. But I prefer this a lot over having something that actually starts with ansibull. During the time the tool was called ansibulled I found that the more complicated tab completion was a much larger PITA than having to remember the `t` :)

Reply:
Thinking about it, this sounds a bit like a reverse engineered argument. If i were able to type the first three letters of a command accident free, i would only have three letter commands and tab completion would have never been invented in the first place.
Actually `ls`, `mv`, `cp`, `rm`, `ln`, `du`, `df`, `dd`, `cc`, `sh`, `su` ... Basic Unix operations are not meant to use tab-completion.
Too bad it's not OK to use `ansible-gendoc`.